processing/src/main/java/io/druid/query/DefaultQueryMetrics.java

```diff
@@ -66,15 +65,6 @@ public void queryType(QueryType query)
   }
 
   @Override
-  public void interval(QueryType query)
```

Comment:
@gianm thanks for your explanation. Your suggestion on interval/duration is exactly same with what I said above. I'll add them if @leventov agrees.